### DISCUSSION TITLE
Accessing Budibase API with Angular webapp

### DISCUSSION THREAD

**Discussion by dpedwards** [Budibase/budibase#5932] [General]
Hi dear BB dev team, 
regarding the information [here](https://budibase.com/blog/building-a-crud-app-with-budibase-and-next.js/) about using the _Budibase_ API endpoints using _Next.js_, surely this should be possible with the _Angular_ front-end web application framework as well. After all, you define REST client(s)/service(s) in the front-end libraries. So you should be able to store the API key from _BB_  in an Angular webapp without any problems or are there any restrictions with _Angular_?

**Comment by shogunpurple** [comment]
There will be no restrictions in angular - it doesn't matter which JS framework, language etc you use - given that the budibase API is just a standard REST API.
